src/DB_Command.php

```diff
@@ -819,6 +828,13 @@ public function size( $args, $assoc_args ) {
 						define( 'TB_IN_BYTES', 1024 * GB_IN_BYTES );
 					}
 
+					if( $human_readable ) {
+						$size_key = floor( log( $row['Size'] ) / log( 1024 ) );
+					    $sizes = array( 'B', 'KB', 'MB', 'GB', 'TB' );
```

Comment:
Those keys are actually base 1000, not base 1024.

See https://github.com/wp-cli/db-command/issues/102 for more information.

For base 1024, you should use the following units:
```suggestion
					    $sizes = array( 'B', 'KiB', 'MiB', 'GiB', 'TiB' );
```

Reply:
@schlessera so based on this, using the `size_format` function that @swissspidy suggested wouldn't work since the values we have are base 1000 and looks like it uses base 1024?

https://core.trac.wordpress.org/browser/tags/5.0/src/wp-includes/functions.php#L239

Reply:
Related: https://core.trac.wordpress.org/ticket/37037, https://core.trac.wordpress.org/ticket/31350

Reply:
> Those keys are actually base 1000, not base 1024.
> 
> See #102 for more information.
> 
> For base 1024, you should use the following units:
> 
> Suggested change Beta 
> [Give us feedback](https://www.research.net/r/SuggestedChanges)
> 					    $sizes = array( 'B', 'KB', 'MB', 'GB', 'TB' );
> 					    $sizes = array( 'B', 'KiB', 'MiB', 'GiB', 'TiB' );

This is not resolved yet. You need to either change the base to 1000, or adapt the units. Otherwise, they are technically incorrect.

Reply:
Sorry thought I got that in the previous commits. I opted for 1000 so we can use more standard "kb" and "mb" etc

Updated now